src/lib/add-on/clients/add-on_auto.rb

```diff
@@ -114,7 +118,7 @@ def change
     end
 
     def export
-      AddOnProduct.Export
+      AddOnProduct.Export.merge(AddOnOthers.Export())
```

Comment:
Why we need yet another module for handling the other addons? Could we simply modify `AddOnProduct.Export`?

Reply:
Replying: No, because we would break the backward compatibility...